### PR TITLE
docs: add faq files

### LINKED ISSUE #4180: [DOCS] addd a `FAQ` overview to the docs/`readme`
## Which page or section is this issue related to?
We got some nice questions during our meetup that can be answered within a FAQ in the docs and readme.

### PR DESCRIPTION
**Pull request by sdiazlor**
# Description

Add the faq document

Closes #4180 

**Type of change**

(Remember to title the PR according to the type of change)

- [ x] Documentation update

**How Has This Been Tested**

(Please describe the tests that you ran to verify your changes.)

- [ x] `sphinx-autobuild` (read [Developer Documentation](https://docs.argilla.io/en/latest/community/developer_docs.html#building-the-documentation) for more details)

**Checklist**

- [ ] I added relevant documentation
- [ ] I followed the style guidelines of this project
- [ ] I did a self-review of my code
- [ ] I made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I filled out [the contributor form](https://tally.so/r/n9XrxK) (see text above)
- [ ] I have added relevant notes to the `CHANGELOG.md` file (See https://keepachangelog.com/)

### CODE REVIEW

**Comment by sdiazlor** [comment]
@davidberenstein1957 I added the faq file, let me know your thoughts. In addition, I was not sure about where to place it, but I thought that maybe they would be useful in getting started.

File: docs/_source/getting_started/faq.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# ❔ FAQ
+
+* *What is Argilla?*
+
+Argilla is an open-source data curation platform for LLMs. Using Argilla, everyone can build robust language models through faster data curation using both human and machine feedback. We provide support for each step in the MLOps cycle, from data labeling to model monitoring. In fact, the inspiration behind the name "Argilla" comes from the word for "clay", in Latin, Italian and even in Catalan. And just as clay has been a fundamental medium for human creativity and tool-making throughout history, we view data as the essential material for sculpting and refining models.
```
Comment (davidberenstein1957) [inline docs/_source/getting_started/faq.md]: maybe you can mention something about "small and large language models"

File: docs/_source/getting_started/faq.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+
+* *Does Argilla train models?*
+
+Argilla is a data curation platform that offers tools to help you train models. With Argilla, you can easily load data and train models straightforward using a feature we call the `ArgillaTrainer`. The `ArgillaTrainer` acts as a bridge to various popular NLP libraries. It simplifies the training process by offering an easy-to-understand interface for many NLP tasks using default pre-set settings without the need of converting data from Argilla's format. You can find more information about training models with Argilla [here](/practical_guides/fine_tune).
```
Comment (davidberenstein1957) [inline docs/_source/getting_started/faq.md]: Maybe explicitely mention that we do not train models but that we do offer integrations for doing so.

File: docs/_source/getting_started/faq.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+
+* *What is the difference between old datasets and the FeedbackDataset?*
+
+The FeedbackDataset stands out for its versatility and adaptability, designed to support a wider range of NLP tasks including those centered on large language models. In contrast, older datasets, while more feature-rich in specific areas, are tailored to singular NLP tasks. For a more detailed explanation, please refer to [this](/practical_guides/choose_dataset) guide.
```
Comment (davidberenstein1957) [inline docs/_source/getting_started/faq.md]: Also mention that the FeedbackDataset is intended for 2.0

File: docs/_source/getting_started/faq.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+
+* *Can Argilla only be used for LLMs?*
+
+No, Argilla is a versatile tool suitable for a wide range of NLP tasks. However, we emphasize the integration with large language models (LLMs), reflecting confidence in the significant role that LLMs will play in the future of NLP. In this page, you can find a list of [supported tasks](/practical_guides/choose_dataset.md#table-comparison).
```
Comment (davidberenstein1957) [inline docs/_source/getting_started/faq.md]: once again "small and large language models"

File: docs/_source/getting_started/faq.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+
+* *Does Argilla provide annotation workforces?*
+
+Currently, we do not offer this service. However, we plan on creating partnerships with annotation providers that ensure ethical practices and secure work environments.
```
Comment (davidberenstein1957) [inline docs/_source/getting_started/faq.md]: we already have partnership. Could you add a CTA(call to action) for people to contact us.

File: docs/_source/getting_started/faq.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+* *How does Argilla differ from competitors like Snorkel, Prodigy and Scale?*
+
+Argilla distinguishes itself for its focus on specific use cases and human-in-the-loop approaches. While it does offer programmatic features, Argilla's core value lies in actively involving human experts in the tool-building process, setting it apart from other competitors.
+
+Furthermore, Argilla places particular emphasis on smooth integration with other tools in the community, particularly within the realms of MLOps and NLP. So, its compatibility with popular frameworks like SpaCy and Hugging Face makes it exceptionally user-friendly and accessible.
+
+Finally, platforms like Snorkel, Prodigy or Scale, while more comprehensive, often require a significant commitment. Argilla, on the other hand, works more as a component within the MLOps ecosystem, allowing users to begin with specific use cases and then scale up as needed. This flexibility is particularly beneficial for users and customers who prefer to start small and expand their applications over time, as opposed to committing to an all-encompassing platform from the outset.
+
+
+* *What is Argilla working on?*
```
Comment (davidberenstein1957) [inline docs/_source/getting_started/faq.md]: "currently working on". Also, maybe we mention distilabel, argilla js and argilla open source

**Comment by davidberenstein1957** [comment]
@sdiazlor , looking good but I had some small remarks.

**Comment by sdiazlor** [comment]
@davidberenstein1957 I made some changes, regarding those, some comments:

* In the docs, I used the dropdown only on the FAQ section as it makes sense to me to have more info on the documentation page and also they can click on the index to go to each section.
* Regarding the README, even if I updated some info and links, I'm not entirely sure about the Cheatsheet section as it looks too long to me (in my opinion, as they are redirected to the documentation, I would just leave the basics avoiding to have it collapsed (as an example for instance)  and then mention the practical guides, tutorials, and end2end examples). Let me know what you think.

**Comment by davidberenstein1957** [comment]
@sdiazlor, there are still some minor things that are not rendered correctly. 
image

**Comment by sdiazlor** [comment]
> @sdiazlor, there are still some minor things that are not rendered correctly.

@davidberenstein1957 Sorry, what things are not rendered?

**Comment by davidberenstein1957** [comment]
> > @sdiazlor, there are still some minor things that are not rendered correctly.
> 
> @davidberenstein1957 Sorry, what things are not rendered?

For me the section that cannot be collapsed look a bit out of place in between all of the options that can be collapsed.

**Review by davidberenstein1957 — APPROVED** [review]

**Pull request merged**